### DISCUSSION TITLE
Παραδοτέο 2 - Π2019119

### DISCUSSION THREAD

**Discussion by ChristosMpazdanhs** [courses-ionio/sw#1238] [Show and tell]
[Αναφορά](https://github.com/ChristosMpazdanhs/sw/tree/2019119/projects/2019119)

**Comment by Fanis-Georg** [comment]
Καλησπέρα συνάδελφε. Θα ήθελα να σου αναφέρω πως το Mint περιέχει systemd οπότε είναι εκτός ύλης το distro που επέλεξες

**Reply by Angeloth1** [reply]
Συνάδελφε όπως αναφέρετε στον πίνακα με τα εβδομαδιαία παραδοτέα αυτήν την εβδομάδα πρέπει να κάνουμε ασκήσεις όντως με ένα σύστημα όπου δεν επιτρέπει customisation ("έχει systemd") όπως και να κάνουμε αυτά που έχουν συζητηθεί σε άλλα discussion για το ομαδικό περιεχόμενο. Από βδομάδα ξεκινάμε την χρήση ενός customisable συστήματος.
